src/main/scala/scala/concurrent/stm/skel/TMapViaClone.scala

```diff
@@ -46,7 +49,10 @@ private[stm] trait TMapViaClone[A, B] extends TMap.View[A, B] with TMap[A, B] {
 
   //////////// builder functionality (from mutable.MapLike via TMap.View)
 
-  override protected[this] def newBuilder: TMap.View[A, B] = empty
+  // ??? in 2.13.0-M4:
+  // overriding method newBuilder in trait View of type => mutable.Builder[(A, B),TMap.View[A,B]]
+  // method newBuilder has incompatible type
+  // override protected[this] def newBuilder: TMap.View[A, B] = empty
```

Comment:
@julienrf do you know what is wrong here?

Reply:
Yes, `mutable.MapOps` doesn’t extend `Builder` anymore. My suggestion here would be to implement `newBuilder` separately in the 2.13- and 2.13+ variants of `TMapViaCloneTemplate`. (And same for `TSetViaCloneTemplate`)